shared-bindings/displayio/Display.c

```diff
@@ -39,7 +39,7 @@
 #include "shared-module/displayio/__init__.h"
 #include "supervisor/shared/translate.h"
 
-//| _DisplayBus = Union[FourWire, ParallelBus, I2CDisplay]
+//| _DisplayBus = Union['FourWire', 'ParallelBus', 'I2CDisplay']
```

Comment:
Why are these needed? I'm new to typing.

Reply:
Quotes around custom types are helpful when the type definitions haven't yet been defined (in this case because the type definitions come later in the same file).  This doesn't bother mypy but pylint complains about this.

Pylint error: "Using variable 'FourWire' before assignment"
...
More Info: https://www.python.org/dev/peps/pep-0484/#forward-references